services/memory.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"github.com/AndresCravioto/academy-go-q42021/loader"
+	"log"
+	"os"
+)
+
+type Stands struct {
+	Store *[]*loader.StandData `json:"store"`
+}
+
+func (b *Stands) Initialize() {
+	filename := "./assets/stands.csv"
+	file, err := os.Open(filename)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	defer file.Close()
+	b.Store = loader.LoadData(file)
```

Comment:
Since we're treating the csv file as a DB I have to advice against doing this preloading and having the all of the data in memory for the following reasons:

* What if the csv has millions and millions of rows with a lot of columns and you don't have enough memory to spare?
* What if the csv get's updated? Then you won't have the most up to date information

It's a lot better to just read the file on each request just like we make queries to DBs when needed, we don't really read all of the info from the DB beforehand